libs/core/src/integration/events.utils.ts

```diff
@@ -0,0 +1,256 @@
+import { ETHERS_BIG_NUMBER, EVM_ADDRESS } from '@hicommonwealth/schemas';
```

Comment:
this should probably go in a future /libs/protocol to keep ethers outside of core - just noting